enhancements/network/cluster-mtu-modification.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Cluster MTU Update
+authors:
+
+- "@mmahmoud"
+
+reviewers:
+
+approvers:
+
+creation-date: 2021-04-12
+status:
+---
+
+# Cluster MTU Update
+
+## Summary
+
+Customers need to modify existing cluster's MTU to enable Jumbo frames, or to
+fix incorrectly provisioned MTU value.
+
+## Motivation
+
+Customer might want to enable Jumbo frames or fix incorrectly provisoned MTU.
+
+## Goals
+
+- Allow customer to change cluster MTU using CRD.
+- Customer will be able to increase or decrease MTU value.
+- no node or pod reload will be required for the MTU to take effect.
+- new nodes comes up should run with the new MTU
+
+## Non goals
+
+- Node's physical NIC's MTU will be handled by the customer.
+- there will be networking outage while MTU changes takes effect, this is
+  will be known, expected and documeneted.
+
+## Proposal
+
+The following diagram shows all places where MTU update need to take effect
+
+![Alt text](./ovn-k8s-node-MTU.png?raw=true)
```

Comment:
I think the diagram is not linked correctly @msherif1234

Reply:
you need to select view file to see the diagram

Reply:
Hmm weird. I thought Rich diff should show it too. NVM.